gradle.properties

```diff
@@ -1,7 +1,9 @@
-projectVersion=2.3.1-SNAPSHOT
+projectVersion=2.4.0.BUILD-SNAPSHOT
```

Comment:
This needs to be `-SNAPSHOT` and not `.BUILD-SNAPSHOT`

Reply:
Done.